app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="row vh-100 d-flex justify-content-center">
+    <div class="col-md-6">
+      <div class="card">
+        <div class="card-body">
+          <h2 class="card-title text-center">Log in</h2>
+
+          <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+            <div class="form-group">
+              <%= f.label :email %><br />
+              <%= f.email_field :email, class: "form-control", autofocus: true, autocomplete: "email" %>
+            </div>
+
+            <div class="form-group">
+              <%= f.label :password %><br />
+              <%= f.password_field :password, class: "form-control", autocomplete: "current-password" %>
+            </div>
+
+            <% if devise_mapping.rememberable? %>
+              <div class="form-group">
+                <%= f.check_box :remember_me %>
+                <%= f.label :remember_me %>
+              </div>
+            <% end %>
+
+            <div class="text-center">
+              <%= f.submit "Log in", class: "btn btn-primary" %>
+            </div>
+          <% end %>
+
+          <%= render "devise/shared/links" %>
+        </div>
```

Comment:
- [ ] Nice work adding devise but there are some issues, firstly you did not include an input for the name in your UI so the user can not fill in his name, and therefore can not submit the sign_up form since the name can not be blank, kindly fix that, secondly, your signup does not look like the wireframe, kindly style properly
![image](https://github.com/MenendezON/Blog-app/assets/66097371/814ef88b-1679-400a-869f-b10a3047357e)
your sign_up page should look similar to the diagram below, all input taking full width with margins by the sides
![Screenshot from 2023-11-18 16-47-34](https://github.com/MenendezON/Blog-app/assets/66097371/224aeb2e-0cef-48ad-bde7-da23f8fffbc9)